SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -574,6 +579,47 @@ void CMeshSolver::UpdateDualGrid(CGeometry *geometry, CConfig *config){
 
 }
 
+void CMeshSolver::ComputeGridVelocity_FromBoundary(CGeometry **geometry, CNumerics **numerics, CConfig *config){
+
+  if (config->GetnZone() == 1)
+    SU2_MPI::Error("It is not possible to compute grid velocity from boundary velocity for single zone problems.\n"
+                   "MARKER_FLUID_LOAD should only be used for structural boundaries.", CURRENT_FUNCTION);
```

Comment:
@Nicola-Fonzi this should prevent this type of grid velocity method from being used when the python wrapper is used.